src/Guzzle6Promise.php

```diff
@@ -63,7 +63,7 @@ public function __construct(PromiseInterface $promise, RequestInterface $request
             }
 
             if (!($reason instanceof GuzzleExceptions\GuzzleException)) {
-                throw new \RuntimeException("Invalid reason");
+                throw new \RuntimeException("Invalid exception returned from Guzzle6", $reason);
```

Comment:
it is not `throw new \RuntimeException("Invalid exception returned from Guzzle6", 0, $reason);`

And what if the reason is not an exception at all (like a string or other thing) ?